src/components/NewBookForm.js

```diff
@@ -7,20 +7,22 @@ const NewBookForm = () => {
       <form>
         <div className="d-flex justify-content-between mediaQueryNewBookForm">
           <div className="form-group col-6 flex-row">
-            <input type="text" className="form-control" placeholder="Book title" />
+            <input
+              type="text"
+              className="form-control"
+              placeholder="Book title"
+            />
           </div>
           <div className="form-group col-3 flex-row">
-            <option>
-              <select
-                name="category"
-                id="category"
-                className="form-select form-control"
-              >
-                <option value="Action">Action</option>
-                <option value="Science Fiction">Science Fiction</option>
-                <option value="Economy">Economy</option>
-              </select>
-            </option>
+            <select
+              name="category"
+              id="category"
+              className="form-select form-control"
+            >
+              <option value="Action">Action</option>
+              <option value="Science Fiction">Science Fiction</option>
+              <option value="Economy">Economy</option>
+            </select>
           </div>
           <div className="form-group col-2 col-sm-3">
```

Comment:
- [ ] [OPTIONAL] As this was not mentioned by the previous reviewer. I kindly suggest that you ensure that your Adding of books is possible. see video below for more information:

https://www.loom.com/share/a431c46f59924682831af5d2a78c720b?sid=cdae3d52-e51e-4856-9891-dab6e449a367